### PR TITLE
JENA-2306: Refactor for easy switch to default JSON-LD 1.1

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-arq/src/main/java/org/apache/jena/riot/SysRIOT.java [hunk 1]
```diff
@@ -113,4 +116,59 @@ public static String filename2baseIRI(String filename) {
...
+    private static ReaderRIOTFactory readerFactoryDft = null;
+    private static ReaderRIOTFactory readerFactoryJSON10;
+    private static ReaderRIOTFactory readerFactoryJSON11;
+
+    /**
+     * <p>
+     * Flip between JSON-LD 1.0 and JSON-LD 1.1 as the default parser for JSON-LD
+     * ({@code Lang.JSONLD}). Both are available as {@code Lang.JSONLD10} and
+     * {@code Lang.JSONLD11} respectively.
+     * This function control the setting of content type
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/riot/SysRIOT.java:131]: s/control/controls

File: jena-arq/src/main/java/org/apache/jena/riot/SysRIOT.java [hunk 2]
```diff
@@ -113,4 +116,59 @@ public static String filename2baseIRI(String filename) {
...
+     * </p><p>
+     * <em>This function is not a permanent API.</em>
+     * </p><p>
+     * The default output is currently fixed as JSON-LD 1.0. A specific version can be obtained by choosing the versioned language name.
+     * </p><p>
+     * Apache Jena uses
+     * <a href="https://github.com/jsonld-java/jsonld-java">jsonld-java</a> for
+     * JSON-LD 1.0 and <a href="https://github.com/filip26/titanium-json-ld">Titanium</a>
+     * for JSON-LD 1.1.
+     * </p><p>We are grateful to each of communities for the work in implementing and maintaining these projects.
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/riot/SysRIOT.java:142]: :+1:

File: jena-arq/src/main/java/org/apache/jena/riot/lang/LangJSONLD11.java [hunk 3]
```diff
@@ -33,48 +34,82 @@
...
         JenaTitanium.convert(dataset, output);
+
+//        try {
+//
+//        } catch (RuntimeException ex) {
+//    }
+//    catch (IOException e) {
+//        errorHandler.error(e.getMessage(), -1, -1);
+//        IO.exception(e) ;
+//    }
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/riot/lang/LangJSONLD11.java:112]: Empty try block, maybe remove?

File: jena-arq/src/main/java/org/apache/jena/riot/system/RiotChars.java [hunk 4]
```diff
@@ -106,9 +106,10 @@ public static boolean isPNChars_U(int ch) {
         return isPNCharsBase(ch) || ( ch == '_' );
     }
 
+    // COnvneience addition.
```
Comment (kinow) [inline jena-arq/src/main/java/org/apache/jena/riot/system/RiotChars.java:109]: s/COnvneience/Convenience

**Review by kinow — APPROVED** [review]

**Comment by afs** [comment]
@kinow  - thanks.

The JSON LD 1.0 /1.1 situation isn't simple! Includes a java bug - Java11 won't talk to schema.org using default java.net.http. Java17 is fine.

**Pull request merged**